Gitagram/View/PostView/PostImageView.swift

```diff
@@ -55,12 +65,24 @@ struct PostImageView: View {
                         .stroke(Color(Color(red: 0.82, green: 0.6, blue: 0.97)), lineWidth: 3)
                 )
             }
-                .onChange(of: selectedPhoto) { selectedPhoto in
-                    Task { await loadImageFromSelectedPhoto(photo: selectedPhoto) }
+            .onChange(of: selectedPhoto) { selectedPhoto in
+                Task {
+                    await loadImageFromSelectedPhoto(photo: selectedPhoto)
```

Comment:
[imo]
これLoadImageFromLibraryUseCaseで実装しても良さそうだね。
```LoadImageFromLibraryUseCase.swift
class LoadImageFromLibraryUseCase {
    func execute(_ pickerItem: PhotosPickerItem) async -> UIImage? {
        if let data = try? await pickerItem?.loadTransferable(type: Data.self) {
            return UIImage(data: data)
        }

        return nil;
    }
}
```

```suggestion
 .onChange(of: selectedPhoto) { selectedPhoto in
        Task { 
            self.image = await LoadImageFromLibraryUseCase().execute(selectedPhoto)
        }
}
```